how-to/how-to-rename-field-in-content-type.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to rename a field in content type
+description: Rename a field in content type using mongodb '$rename' operator
+type: how-to
+page_title: Rename a field in a content type
+slug: rename-field-in-content-type
+position:
+tags: platform-core, rename-field, backendservices, content-type
+teampulseid: 
+ticketid:  1046311
+pitsid:
+aws: Amazon Web Services
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Telerik Platform General</td>   
+  </tr>
+</table>
+
+## Description
+You want to rename an existing field in a content type in {{site.tp}}. The rename option is not available not available from {{site.tp}} UI. 
+
+## Solution
+Renaming a field is not available in {{site.tp}} UI as it will introduce changes in the database data which is an advanced operation and should be performed with caution. 

```

Comment:
There is no need to explain why it is not available. It would be better to explain the case - You can rename a field of the data items by....